src/main/java/org/jenkinsci/plugins/github/extension/GHEventsSubscriber.java

```diff
@@ -73,6 +73,9 @@ protected void onEvent(GHEvent event, String payload) {
         return new Function<GHEventsSubscriber, Set<GHEvent>>() {
             @Override
             public Set<GHEvent> apply(GHEventsSubscriber subscriber) {
+                if (subscriber == null) {
```

Comment:
this never should happen. Use Validate.notNull() from lang3 - it should throw exc and fixed if that happens